composer/core/state.py

```diff
@@ -416,6 +416,7 @@ def __init__(
         self._train_dataloader = train_dataloader
         self._evaluators = list(ensure_tuple(evaluators))
 
+        self.previous_timestamp: Optional[Timestamp] = None
```

Comment:
Thoughts on instead starting it as `timestamp` instead of `None`? Does this make anything easier?